pkg/volume/fc/fc_test.go

```diff
@@ -53,6 +53,9 @@ func TestCanSupport(t *testing.T) {
 	if plug.CanSupport(&volume.Spec{Volume: &v1.Volume{VolumeSource: v1.VolumeSource{}}}) {
 		t.Errorf("Expected false")
 	}
+	if plug.CanSupport(&volume.Spec{}) {
```

Comment:
Also test case for PersistentVolumeSpec?

Reply:
`(&volume.Spec{}` means spec.Volume is nil , and spec.PersistentVolume is nil too

Reply:
Yes I mean more test cases showing success and failure if you specify a PersistentVolumeSource

Reply:
And success test case for specifying VolumeSource

Reply:
@msau42 Done! PTAL